integration/common.sh

```diff
@@ -23,23 +23,46 @@ function runTest() {
 function waitForServerStartup() {
   # The server will 404 the request as there's no handler for it. This error doesn't matter
   # as the test will fail if the server is really not up.
-  PORT=$1
+  local PORT=$1
```

Comment:
nit: lowercase "port"?

(+1 for local)

https://google.github.io/styleguide/shell.xml#Variable_Names